src/main/java/commands/inputParser.java

```diff
@@ -0,0 +1,115 @@
+package commands;
+
+import exceptions.AragornException;
+
+public class inputParser {
+    private String[] splitInput = new String[3];
```

Comment:
avoid using magic literals in the code, consider storing it in a constant (static final) 

https://nus-cs2113-ay2324s2.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-magic-numbers